src/expr.c

```diff
@@ -4450,7 +4450,11 @@ int sqlite3ExprCodeTarget(Parse *pParse, Expr *pExpr, int target){
       }
       pCol = &pAggInfo->aCol[pExpr->iAgg];
       if( !pAggInfo->directMode ){
+        #ifdef FREEBSD_KERNEL
+        //todo: STELIOS
```

Comment:
@stelkasouridis : It seems like you should emit some warning here that the code isn't ready yet.